content/en/user-guide/aws/verifiedpermissions/index.md

```diff
@@ -0,0 +1,423 @@
+{{< /command >}}
+
+The above command returns the following response:
+
+```json
+{
+    "policyStoreId": "q5PCScu9qo4aswMVc0owNN",
+    "arn": "arn:aws:verifiedpermissions::000000000000:policy-store/q5PCScu9qo4aswMVc0owNN",
+    "createdDate": "2025-04-22T19:24:11.175557Z",
+    "lastUpdatedDate": "2025-04-22T19:24:11.175557Z"
+}
+```
+
+You can list all the Verified Permissions policy stores using the [`ListPolicyStores`](https://docs.aws.amazon.com/verifiedpermissions/latest/apireference/API_ListPolicyStores.html) API.
+Run the following command to list all the Verified Permissions policy stores:
+
+{{< command >}}
+$ awslocal verifiedpermissions list-policy-stores
+{{< /command >}}
+
+### Create a Policy
+
+To create a Verified Permissions Policy, use the [`CreatePolicy`](https://docs.aws.amazon.com/verifiedpermissions/latest/apireference/API_CreatePolicy.html) API.
+
+First, create a JSON file containing the following policy named `static_policy.json`:
+
+```json
+{
+    "static": {
+        "description":  "Grant the User alice access to view the trip Album",
+        "statement": "permit(principal == User::\"alice\", action == Action::\"view\", resource == Album::\"trip\");"
+    }
+}
+```
+
+You can then run this command to create the policy:
+{{< command >}}
+$ awslocal verifiedpermissions create-policy \
+    --definition file://static_policy.json \
+    --policy-store-id q5PCScu9qo4aswMVc0owNN
+{{< /command >}}
+
+Replace the policy store ID with the ID of the policy store you created previously.
+
+You should see the following output:
+
+```json
+{
+    "policyStoreId": "q5PCScu9qo4aswMVc0owNN",
+    "policyId": "MfsIseJDeZsr5WUm3tB4FX",
+    "policyType": "STATIC",
+    "principal": {
+        "entityType": "User",
+        "entityId": "alice"
+    },
+    "resource": {
+        "entityType": "Album",
+        "entityId": "trip"
+    },
+    "actions": [
+        {
+            "actionType": "Action",
+            "actionId": "view"
+        }
+    ],
+    "createdDate": "2025-04-22T19:25:25.161652Z",
+    "lastUpdatedDate": "2025-04-22T19:25:25.161652Z",
+    "effect": "Permit"
+}
+```
+
+### Authorize a request
+
+We can now make use of the Policy Store and the Policy to start authorizing requests.
+To authorize a request using Verified Permissions, use the [`IsAuthorized`](https://docs.aws.amazon.com/verifiedpermissions/latest/apireference/API_IsAuthorized.html) API.
+
+{{< command >}}
+$ awslocal verifiedpermissions is-authorized \
+  --policy-store-id q5PCScu9qo4aswMVc0owNN \
+  --principal entityType=User,entityId=alice \
+  --action actionType=Action,actionId=view \
+  --resource entityType=Album,entityId=trip
+{{< /command >}}
+
+You should get the following output, indicating that your request was allowed:
+
+```json
+{
+    "decision": "ALLOW",
+    "determiningPolicies": [
+        {
+            "policyId": "MfsIseJDeZsr5WUm3tB4FX"
+        }
+    ],
+    "errors": []
+}
+```
+
+## Integration with Cognito
```

Comment:
I'm just wondering if we need a specific section on Cognito integration. Are we documenting some specialized setup that works on LocalStack, but may not work on AWS? If the implementation is straightforward and in parity with AWS, then we don't need to have a specific section on this aspect. What do you think?

Reply:
Good question. It is somewhat straightforward and the same as AWS, but I used the documentation for now to showcase this is possible to do. I think testing regular policies could be done using Cedar itself, but the custom AWS integration with Cognito and us supporting it is what is "bringing value". This could also be a proper sample maybe?
If you think it does not make sense, I'm fine with removing it

Also, there wasn't really a proper tutorial online to set this up even for AWS, otherwise I would have sent the users towards it.